devops/openshift/_legal-api.dc.json

```diff
@@ -109,23 +455,22 @@
                         "type": "ConfigChange"
                     }
                 ],
-                "replicas": "${REPLICAS}",
+                "replicas": "${{REPLICA_MIN}}",
                 "test": false,
                 "selector": {
-                    "app": "${COMP_NAME}-${SUFFIX}",
-                    "deploymentconfig": "${COMP_NAME}-${SUFFIX}"
+                    "name": "${COMP_NAME}-${SUFFIX}"
                 },
                 "template": {
                     "metadata": {
+                        "name": "${COMP_NAME}-${SUFFIX}",
                         "labels": {
-                            "app": "${COMP_NAME}-${SUFFIX}",
-                            "deploymentconfig": "${COMP_NAME}-${SUFFIX}"
+                            "name": "${COMP_NAME}-${SUFFIX}"
                         }
                     },
                     "spec": {
                         "containers": [{
                             "name": "${COMP_NAME}",
-                            "image": "docker-registry.default.svc:5000/${IMAGE_NAMESPACE}/${COMP_NAME}:${SUFFIX}",
+                            "image": " ",
```

Comment:
Is this on purpose? Will it create the container without an image somehow?

Reply:
This is on purpose yes - the groovy script that actually facilitates the deployment populates that value with properties derived from other variables.